src/Taxjar/DependencyInjection/TaxjarServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,37 @@
+#if NETSTANDARD2_0
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+using System;
+
+using Taxjar.Options;
+
+namespace Taxjar.DependencyInjection
+{
+    /// <summary>
+    ///  Add TaxjarApi to Dependency Injection.
+    /// </summary>
+    public static class TaxjarServiceCollectionExtensions
+    {
+        public static IServiceCollection AddTaxjar(this IServiceCollection services,
+                                                   Action<TaxjarOptions> configure = default)
+        {
+
+            services.AddChangeTokenOptions(nameof(TaxjarOptions), string.Empty, configure);
```

Comment:
Since this is dependent on the `BET.Extensions.Options` library, this line will need to be removed.

Reply:
Yes, I can replace with AddOptions<> to register this.